### PR TITLE
Added a save icon in the save button

### LINKED ISSUE #3574: [UI] Settings: Labels overlapping text in input fields
#### Description

Field labels overlap with field input, specifically saved user preference input.

When a user changes the performance load test defaults from the system defaults, the UI will correctly save the user's preference, but will display their preferred settings as overlapping the label of the input field.

#### Expected Behavior
1. Move the field label out of the actual text field and move to a wrapping label (see the Duration field label(
2. User preference values should not overlap the field label.

#### Screenshots/Logs
Screen Shot 2021-07-20 at 12 42 04 PM

#### Environment

- Host OS: Mac
- Meshery Server Version: stable-v0.5.37

---

**Contributor Resources:**
- (Contributing Docs](https://docs.meshery.io/project/contributing)

### PR DESCRIPTION
**Pull request by sayantan1413**
Signed-off-by: sayantan1413 <sayantanbose2001@gmail.com>

**Description**

Added a save icon in the save button

This PR fixes #3574 

**Notes for Reviewers**

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by sayantan1413** [comment]
Screenshot 2021-07-30 at 1 16 55 AM

File: ui/components/MesherySettingsPerformanceComponent.js [hunk 1]
```diff
@@ -2,6 +2,7 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import Button from '@material-ui/core/Button';
+import SaveIcon from '@material-ui/icons/Save';
```
Comment (warunicorn19) [inline ui/components/MesherySettingsPerformanceComponent.js:5]: Hey @sayantan1413, i think we are supposed to use the non-filled save icon. The save icon you have used is a filled one as far as i know, try importing 'SaveOutlinedIcon' instead

**Review by warunicorn19 — CHANGES_REQUESTED** [review]

File: ui/components/MesherySettingsPerformanceComponent.js (hunk 1 above)
Comment (sayantan1413) [reply ui/components/MesherySettingsPerformanceComponent.js:5]: Screenshot 2021-07-30 at 1 24 41 AM

You mean this one right?

Comment (warunicorn19) [reply ui/components/MesherySettingsPerformanceComponent.js:5]: yes

**Review by warunicorn19 — COMMENTED** [review]
LGTM, thanks @sayantan1413. would you also like to fix the overlapping text?
This is the workaround that @sudo-NithishKarthik suggested
https://material-ui.com/components/text-fields/#limitations

File: ui/components/MesherySettingsPerformanceComponent.js (hunk 1 above)
Comment (leecalcote) [reply ui/components/MesherySettingsPerformanceComponent.js]: Thank you, gents. A little more space between the icon and word, would be great.

**Review by leecalcote — DISMISSED** [review]

**Comment by sayantan1413** [comment]
Yes sure @warunicorn19 I will try this out.

**Comment by sayantan1413** [comment]
Screenshot 2021-07-30 at 1 40 10 AM

**Review by warunicorn19 — APPROVED** [review]
LGTM, Thanks @sayantan1413 .

**Review by warunicorn19 — APPROVED** [review]

**Review by warunicorn19 — DISMISSED** [review]

**Comment by sayantan1413** [comment]
There are some more changes regarding the overlapping text change I need to do.

**Comment by sayantan1413** [comment]
Screenshot 2021-07-30 at 2 55 45 AM

**Comment by leecalcote** [comment]
@sayantan1413 while we're being picky about this layout, do you mind making the ? next to Load Generators something of a superscript? (making about half the size it is now)?

**Comment by sayantan1413** [comment]
Yeah sure @leecalcote I will make it half the size😃

**Comment by sayantan1413** [comment]
Screenshot 2021-07-30 at 10 52 13 AM

Does this look good @leecalcote ?

File: ui/components/MesherySettingsPerformanceComponent.js (hunk 1 above)
Comment (sayantan1413) [reply ui/components/MesherySettingsPerformanceComponent.js]: Yes @leecalcote sure I have added a space between the icons and the text.

**Comment by warunicorn19** [comment]
> Screenshot 2021-07-30 at 11 00 22 AM
> 
> Does this look good @leecalcote ?

A little too small don't you think? Looks like a copyright symbol.

**Comment by leecalcote** [comment]
> > Screenshot 2021-07-30 at 11 00 22 AM
> > Does this look good @leecalcote ?
> 
> A little too small don't you think? Looks like a copyright symbol.

Feel free to play around a little. This is the first of its kind. It might be that it should be that size, but not positioned like superscript - just vertically-centered in the middle of the line.

**Comment by sayantan1413** [comment]
Yeah @leecalcote I will play with this a bit.

**Comment by leecalcote** [comment]
Remove the PR/Draft label when you are ready for review again.

**Comment by sayantan1413** [comment]
Screenshot 2021-08-03 at 7 30 46 PM
Hey @warunicorn19 how does it look now?

**Comment by warunicorn19** [comment]
@sayantan1413 yeah this looks good to me.

**Comment by sayantan1413** [comment]
Hey @leecalcote I think this look better now 🤔

**Comment by sudo-NithishKarthik** [comment]
@sayantan1413 , are we showing any tooltip when the user hovers over it ?

**Comment by sayantan1413** [comment]
Yes @sudo-NithishKarthik while hovering over it there is a tooltip.

**Review by warunicorn19 — APPROVED** [review]
Looks great to me! thanks @sayantan1413

**Review by leecalcote — APPROVED** [review]
Let's see how this plays... Thank you, @sayantan1413

**Pull request merged**